saltswap/tests/test_recorder.py

```diff
@@ -85,3 +87,5 @@ def test_write_read(self):
         for var, res in zip(sample_variables, saved_results):
             assert np.all(opened_ncfile.groups['Sample state data'][var][:] == res)
 
+        remove(filename)
```

Comment:
The netcdf file is closed before removing it (on [line 81](https://github.com/choderalab/saltswap/blob/80de64c220d76508db5224811f0ede75ba2b4ab8/saltswap/tests/test_recorder.py#L81) and the test does run without error.